### DISCUSSION TITLE
Hero 11 MINI- Overlay issue

### DISCUSSION THREAD

**Discussion by Knicks325** [gopro/labs#681] [General]
I would like to add the date and time as an overlay on the Hero 11 MINI - is that possible - I know it works on the Hero 11 and I just got the MINI but I am having issues. Please let me know.

**Comment by dnewman-gpsw** [comment]
There is a firmware release tomorrow with small fix for overlay, although it does work on HERO11 Mini today.
![image](https://github.com/gopro/labs/assets/10856084/6f855834-1cf0-43d9-a6b5-0537fd65393a)

Set you resolution to 4K30/25 or 24, 8-bit.  The bug today requires you to set the font size to 30 pixel minimum (should be fixed in tomorrows release.)

**Reply by Knicks325** [reply]
it worked thank you!!!!

**Comment by Knicks325** [comment]
Am I able to set the Time to army time?

**Reply by dnewman-gpsw** [reply]
Yes. Controlled here https://gopro.github.io/labs/control/overlays/
